src/main/utils/_initialSettings.js

```diff
@@ -3,4 +3,5 @@ export default {
   enableJSONApi: true,
   nativeFrame: (process.platform !== 'win32'),
   enableWindowTitleInfo: false,
+  auto-launch: false,
```

Comment:
You can't have a `'-'` inside an unquoted object key.

Please change this to 

`"auto-launch": false,`

In future please run `npm run lint && npm run build && npm start` to ensure everything works